app/js/components/manageApps/StartErrorModal.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import {
+  Modal, ModalHeader, ModalTitle,
+  ModalClose, ModalBody, ModalFooter
+} from 'react-modal-bootstrap';
+
+export default class StartErrorModal extends Component {
+  render() {
+    const {
+      isOpen,
+      hideMsgModal,
+      message,
+      markAsRead,
+      app,
+    } = this.props;
+    return (
+      <Modal isOpen={isOpen} onRequestHide={hideMsgModal}>
+        <ModalHeader>
+          <ModalClose onClick={hideMsgModal} />
+          <ModalTitle>
+            <span id="msg-modal-title">
+              <span className="glyphicon glyphicon-warning-sign" id="error-title-logo" />
+              {message}{app.name}
+            </span>
+          </ModalTitle>
+        </ModalHeader>
+        <ModalBody>
+          <p>{app.startupErrorMessage}</p>
+        </ModalBody>
+        <ModalFooter>
+          <button
+            className="btn btn-default btn-mark-read"
+            onClick={markAsRead}
+          >
+            Mark as Read
+          </button>
+        </ModalFooter>
+      </Modal>
+    );
+  }
+}
+
+StartErrorModal.propTypes = {
```

Comment:
`React.PropTypes` is deprecated. You should install `prop-types` as a separate dependency.

Reply:
👍